app/helpers/catalog_helper/textual_summary.rb

```diff
@@ -9,9 +10,16 @@ def tags_from_record
         p[:value].push(value)
       else
         name = Classification.find_by(:id => Classification.find_by(:tag_id => tag.id).parent_id).description
-        tags.push(:icon => "fa fa-tag", :label => name, :value => [value])
+        if tag_categories[name].nil?
+          tag_categories[name] = [value]
+        else
+          tag_categories[name].append(value)
+        end
```

Comment:
...this would become

```suggestion
        tag_categories[name] << value
```

Reply:
The if branch is to prevent it from throwing errors due to appending values to an undefined value. The way I had it initially set up tag_categories[name] was initially undefined so when I tried appending values to it it would throw an error, so I added the if branch to initialize the array for the first value